Jellyfin.Server.Implementations/Metrics/UserMetrics.cs

```diff
@@ -0,0 +1,281 @@
+#pragma warning disable CA1307
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Jellyfin.Data;
+using Jellyfin.Data.Events;
+using Jellyfin.Database.Implementations.Entities;
+using Jellyfin.Database.Implementations.Enums;
+using MediaBrowser.Controller.Authentication;
+using MediaBrowser.Controller.Library;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Prometheus;
+
+namespace Jellyfin.Server.Implementations.Metrics.UserMetrics
+{
+    /// <summary>
+    /// Provides Prometheus metrics related to Jellyfin users.
+    /// </summary>
+    public class UserMetrics : IHostedService
+    {
+        private readonly ILogger<UserMetrics> _logger;
+        private readonly IEnumerable<IAuthenticationProvider> _authenticationProviders;
+        private readonly IUserManager _userManager;
+
+        // Prometheus Gauges for user metrics
+        private static readonly Gauge _totalUserAccounts = Prometheus.Metrics
+            .CreateGauge("jellyfin_user_accounts_total", "Total number of user accounts in Jellyfin");
+
+        private static readonly Gauge _activeUserAccounts = Prometheus.Metrics
+            .CreateGauge("jellyfin_user_accounts_active", "Number of active (non-disabled) user accounts");
+
+        private static readonly Gauge _adminUserAccounts = Prometheus.Metrics
+            .CreateGauge("jellyfin_user_accounts_admin", "Number of administrator user accounts");
+
+        private static readonly Gauge _usersByAuthProvider = Prometheus.Metrics
+            .CreateGauge("jellyfin_users_by_auth_provider", "Number of users by authentication provider", new[] { "provider" });
+
+        private static readonly Gauge _usersByPermission = Prometheus.Metrics
+            .CreateGauge("jellyfin_users_by_permission", "Number of users with specific permissions", new[] { "permission" });
+
+        private static readonly Gauge _usersWithExternalAuth = Prometheus.Metrics
+            .CreateGauge("jellyfin_users_external_auth_total", "Total number of users using external authentication");
+
+        private static readonly Gauge _recentlyActiveUsers = Prometheus.Metrics
+            .CreateGauge("jellyfin_users_recently_active", "Number of users active within the last 30 days");
```

Comment:
This is also not needed as one can just do: `sum(jellyfin_sessions_total{state="active"})`